Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest
+
+# Since Alpine linux 3.3 we can install on-the-fly
+RUN apk add --no-cache perl 
```

Comment:
That's simple enough :-)

Could you remove the trailing whitespace in this line ? Also, do you think the comment is useful ?

I would like a comment on why using alpine rather than official perl image ?

Reply:
For the comment it's just for justify the non usage of `rm -rf /var/cache/apk/*` but it's not mandatory.
I prefer to use alpine because I just want a light version not a `debian:stretch` base

Reply:
Ok, on verra si ça pose problème avec la libc d'alpine.